api/events/msam-events.json

```diff
@@ -38,7 +38,8 @@
                                 "source": [
                                     "aws.medialive",
                                     "aws.mediapackage",
-                                    "aws.mediastore"
```

Comment:
We're actually not using this intermediary file anymore. We go from the base yml file to the release version of this template. So we probably should remove this from the repo actually.